app/validation.py

```diff
@@ -136,9 +138,13 @@ def get_validation_errors(validator_name, json_data,
             error_map[key] = _translate_json_schema_error(
                 key, error.validator, error.validator_value, error.message
             )
-        elif error.validator == 'required':
-            key = re.search(r'\'(.*)\'', error.message).group(1)
+        elif error.validator in ['required', 'dependencies']:
+            regex = r'u?\'(\w+)\' is a dependency of u?\'(\w+)\'' if error.validator == 'dependencies' else r'\'(.*)\''
```

Comment:
<sub>Obligatory comment on json schema errors being unhelpful</sub>
Do we get an error message for each of the missing keys? E.g. if we have `["agileCoachMinPrice", "agileCoachMaxPrice"]` in the dependency and both are missing – will we get `answer_required` error for both or only for one of them?